app/controllers/hyrax/embargoes_controller.rb

```diff
@@ -1,5 +1,56 @@
 module Hyrax
   class EmbargoesController < ApplicationController
-    include Hyrax::EmbargoesControllerBehavior
+    include Hyrax::Collections::AcceptsBatches
+
+    def index
+      authorize! :index, Hydra::AccessControls::Embargo
+    end
+
+    def edit
+      work = find_resource(params[:id])
+      authorize! :edit, work
+      @resource = find_resource(work.embargo_id)
+      @change_set = EmbargoChangeSet.new(@resource, parent_resource: work)
+    end
+
+    # Removes a single embargo
+    def destroy
+      work = find_resource(params[:id])
+      authorize! :destroy, work
+      embargo = Hyrax::Actors::EmbargoActor.new(work).destroy
+      flash[:notice] = embargo.embargo_history.last
+      if work.work? && work.file_sets.present?
+        redirect_to confirm_permission_path
+      else
+        redirect_to edit_embargo_path
```

Comment:
Will you need parameters for this? Most single resources require an ID

Reply:
I'm just moving this code from a module to a class.